internal/rpc/rpcserver/server.go

```diff
@@ -1814,6 +1814,45 @@ func (s *walletServer) RevokeTickets(ctx context.Context, req *pb.RevokeTicketsR
 	return &pb.RevokeTicketsResponse{}, nil
 }
 
+func (s *walletServer) RevokeTicket(ctx context.Context, req *pb.RevokeTicketRequest) (*pb.RevokeTicketResponse, error) {
+	var ticketHash *chainhash.Hash
+	var err error
+	if len(req.TicketHash) != 0 {
```

Comment:
You also need to return early from this call in an `else` block, otherwise you'll end up passing a nil `ticketHash` to `RevokeTicket`